tutorials/horizontally-scale-mysql-database-backend-with-google-cloud-sql-and-proxysql/index.md

```diff
@@ -0,0 +1,325 @@
+
+## Creating Read Replicas
+
+Note: The following steps use [Cloud Console](https://console.cloud.google.com/sql) to create a Cloud SQL read replica. You can also create a read replica [via Cloud SDK](https://cloud.google.com/sdk/gcloud/reference/sql/instances/create) or [using the Cloud SQL API](https://cloud.google.com/sql/docs/mysql/admin-api/v1beta4/). 
+
+1. Go to the [Cloud SQL Instances page](https://console.cloud.google.com/sql/instances) in the Cloud Console. Click the instance you just created. Select the **Replicas** tab.
+2. Click **Create read replica**. Enter an Instance ID and pick a location for the replica. If you would like to, click **Show configuration options** to give the replica a different machine type and storage solution from the master instance. Settings such as authorized networks are inherited from the master instance, though you can still change them specifically for the read replica.
+3. Click **Create**. It may take a while to create the instance and replicate the data.
+4. After the read replica comes online, click its name and write down the **IPv4 address** in the **Connect to this instance** card of its Instance Details page.
+5. Repeat the steps once to create another read replica. Write down its IPv4 address as well.
+
+See [Creating Read Replicas](https://cloud.google.com/sql/docs/mysql/replication/create-replica) and [Requirements and Tips for Configuring Replication](https://cloud.google.com/sql/docs/mysql/replication/tips#read-replica) for more information about Cloud SQL read replicas.
+
+## Configuring ProxySQL to Distribute Database Queries
+
+### Adding Read Replicas
+
+In the ProxySQL admin interface, run the following SQL queries:
+
+```
+INSERT INTO mysql_servers (hostgroup_id, hostname) VALUES ('1', '[IP_ADDRESS_#1]'), ('1', '[IP_ADDRESS_#2]')
+LOAD MYSQL SERVERS TO RUNTIME
+SAVE MYSQL SERVERS TO DISK
+```
+
+Replace `[IP_ADDRESS_#1]` and `[IP_ADDRESS_#2]` with the IPv4 addresses of the two read replicas you just created. Note that the replicas are assigned to a different group (`1`) from the master instance (`0`).
+
+### Updating Rules for Distribution
+
+Note: Steps below set a simple list of rules that distribute all write traffic and 20% read traffic to the master instance, 40% read traffic to the first read replica and another 40% read traffic to the second read replica. It is based on [the sample rules in ProxySQL wiki](https://github.com/sysown/proxysql/wiki/ProxySQL-Read-Write-Split-(HOWTO)#basic-readwrite-split-using-regex). **Those rules are for demonstration purposes only and may not fit your application well.** Allocate different weights to read replicas as you fit and experiment with other query routing functionalities [(regex, digest, etc.)](https://github.com/sysown/proxysql/wiki/ProxySQL-Read-Write-Split-(HOWTO)#readwrite-split-using-regex-and-digest) provided in ProxySQL to create context-sensitive rules for better query distribution.
```

Comment:
Suggested wording: "**Those rules are for demonstration purposes only and may not suit the performance requirements of your particular application.**"